docker/worker/worker.py

```diff
@@ -322,6 +322,9 @@ def filter_unsupported_ecosystems(vulnerability):
   for affected in vulnerability.affected:
     if osv.ecosystems.get(affected.package.ecosystem):
       filtered.append(affected)
+    # CVE-converted OSV records have no package information.
+    if affected.package.ecosystem == '':
```

Comment:
`affected.HasField('package')` communicates the intention of what you're trying to do much better. i.e. allow in cases where there is just no `package` specified to begin with. 

`affected.HasField('package')` works differently than your check: it returns false only if no fields in the nested structure are set. In the example I gave, it would return true because one of the fields is set, which fails the check.

Reply:
Got it, thanks for clarifying. Addressed in https://github.com/google/osv.dev/pull/925/commits/2ecf0d4443aa6fd0670d1258bb44e0502cc25be4